docs/UserGuide.md

```diff
@@ -44,65 +44,70 @@ This user guide provides in-depth descriptions of all the available features in
+   * **`find`** `John Doe` : Finds all the guests who have at least one field containing `John` or `Doe` (_case-insensitive_).
 
-   * **`find`** `John Doe` : Finds all the guests where at least one of the fields contains `John` or `Doe` (case-insensitive).
+8. To make changes to `John Doe`, you can use the **`edit`** command. This changes the guest's fields based on the new values you provide. The other values will remain unchanged. <br>
+For example, if `John Doe` is the second guest in the list, and you want to change his room number to `05-55`, you can do this.
+    * **`edit`** `2` `rm/05-55`n: Changes the `Room` of `John Doe` to `05-55` in GuestBook.
 
-8. To make changes to `John Doe`, you can use the **`edit`** command. This changes the guest's fields based on the values you have inputted. The other values will remain the same. <br> For example, if `John Doe` is the second guest in the list, and you want to change his room number to 05-55, you can do this.
-    * **`edit`**`2` `rm/05-55`n: Changes the `Room` of `John Doe` to `05-55` in GuestBook.
+9. After executing a **`find`** command, you might realise that some of your guests are no longer visible, as they are not in the current list.<br>
+To see a list of all your guests, you can use the **`list`** command.
+     * **`list`**: List all the guests in GuestBook.
 
-9. After running a **`find`** function, you might realise that some of your guests are no longer visible, as they are not in the current list. To see a list of all your guests, you can use the **`list`** command.
-    * **`list`**: List all the guests in GuestBook.
+10. In the event that a guest orders room service, you can add charges to the guest's bill using the **`bill`** command.<br>
+For example, you can charge the third guest `30.35` by running the command below.
+    * **`bill`** `3` `b/+30.35`: Adds `30.35` to the bill of the third guest.
 
-10. In the situation that a guest purchases additional necessities, you can bill the guest using the **`bill`** command. For example, you can bill the third guest a service that costs 30.35 by running the command below.
-    * **`bill`** `3` `b/+30.35`: Adds 30.35 to the bill of the third guest.
+11. When a guest checks out of your hotel, you can remove the guest using the **`delete`** command.
+    * **`delete`** `3` : Deletes the third guest shown in the current list.
 
-11. When a guest checkouts, you might want to delete the guest. In that case, you can use the **`delete`** command.
-    * **`delete`**`3` : Deletes the third guest shown in the current list.
-
-12. To clear your GuestBook of all entries, you can use the **`clear`** command. Do be warn that it is irreversible, so use with caution!
+12. To clear your GuestBook of all entries, you can use the **`clear`** command.<br>
+<span style="background-color: yellow; color: black">**WARNING:** This action is **irreversible**. Use with caution.</span>
```

Comment:
This line is repeated under features, clearing all entries, should we have it here as well? Or do you think its necessary?

Reply:
I think it's good to leave it here as the user might accidentally unintentionally clear the data while following the tutorial!